aspnetcore/fundamentals/metapackage.md

```diff
@@ -33,3 +36,35 @@ You can use the package trimming process to remove packages that you don't use.
 The following *.csproj* file references the `Microsoft.AspNetCore.All` metapackage for ASP.NET Core:
 
 [!code-xml[](../mvc/views/view-compilation/sample/MvcRazorCompileOnPublish2.csproj?highlight=9)]
+
+::: moniker range=">= aspnetcore-2.1"
+# Microsoft.AspNetCore.All metapackage for ASP.NET Core 2.1 and later version range locking
+
+`Microsoft.AspNetCore.All` 2.1 and later specifies package dependency ranges. The package dependency ranges ensure that only the excact version specified for each dependency is allowed. Any other dependency that attempts to lift a version for a component beyond what is specified in the meta-package will result in a NuGet warning, which the Microsoft.NET.Sdk will elevate to an error by default. 
+
+::: moniker-end
+
+
+<a name="migrate"></a>
+## Migrating from Microsoft.AspNetCore.All to Microsoft.AspNetCore.App
+
+The following packages are direct dependencies of `Microsoft.AspNetCore.All` but not the `Microsoft.AspNetCore.App` package. 
+
+* Microsoft.Data.Sqlite
```

Comment:
How did you generate this list?

I took a diff of the packages and these are the extra packages in .All and not in .App.
```
Microsoft.AspNetCore.ApplicationInsights.HostingStartup
Microsoft.AspNetCore.AzureAppServices.HostingStartup
Microsoft.AspNetCore.AzureAppServicesIntegration
Microsoft.AspNetCore.DataProtection.AzureKeyVault
Microsoft.AspNetCore.DataProtection.AzureStorage
Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv
Microsoft.AspNetCore.SignalR.Redis
Microsoft.Data.Sqlite
Microsoft.Data.Sqlite.Core
Microsoft.EntityFrameworkCore.Sqlite
Microsoft.EntityFrameworkCore.Sqlite.Core
Microsoft.Extensions.Caching.Redis
Microsoft.Extensions.Configuration.AzureKeyVault
Microsoft.Extensions.Logging.AzureAppServices
Microsoft.VisualStudio.Web.BrowserLink
```

A few things to note. Microsoft.AspNetCore.Identity.Service.AzureKeyVault is not a shipping package and Microsoft.AspNetCore.AzureKeyVault.HostingStartup only exists in 2.0 and is not (and was never) included in the .All metapackage.